setup.py

```diff
@@ -74,7 +74,7 @@ def run(self):
     python_requires=">=3.6",
     setup_requires=["setuptools_scm"] + sphinx_requirements,
     install_requires=[
-        "dask[dataframe,distributed]==2021.10.0",
+        "dask[dataframe,distributed]>=2021.10.0",
```

Comment:
We'll need this if we want to use unpinned versions of Dask with the nightlies for certain features (`dask-sql-server` in particular)